src/app/core/services/si/si-core/intacct-connector.service.ts

```
@@ -18,13 +23,34 @@
     private apiService: ApiService,
     private workspaceService: SiWorkspaceService,
-    private workspace: SiWorkspaceService,
-    private storageService: StorageService
+    private storageService: StorageService,
+    private mappingsService: SiMappingsService,
+    private toastService: IntegrationsToastService,
+    private translocoService: TranslocoService
   ) { }
 
+  mapAPIResponseToSage300ConnectorFormGroup(sageIntacctConnection: SageIntacctCredential | null): FormGroup {
+    const isDisabled = sageIntacctConnection?.si_company_id ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)